### PR TITLE
[REVIEW] Fixed negative float

### PR DESCRIPTION
**Pull request by BradReesWork**

### CODE REVIEW

**Comment by ogreen** [comment]
This looks good to me.

File: cudf/tests/test_csvreader.py [hunk 1]
```diff
@@ -144,3 +144,26 @@ def test_csv_reader_skiprows_skipfooter(tmpdir):
     assert len(out.columns) == len(df_out.columns)
     assert len(out) == len(df_out)
     pd.util.testing.assert_frame_equal(df_out, out.to_pandas())
+
+
+def test_csv_reader_negative_vals(tmpdir):
+    fname = 'tmp_csvreader_file8.csv'
```
Comment (kkraus14) [inline cudf/tests/test_csvreader.py]: Can you follow the way we specify filename / path in the previous tests?

File: cudf/tests/test_csvreader.py [hunk 2]
```diff
@@ -144,3 +144,26 @@ def test_csv_reader_skiprows_skipfooter(tmpdir):
...
+
+    names = ['0', '1', '2']
+    dtypes = ['float', 'float', 'float']
```
Comment (kkraus14) [inline cudf/tests/test_csvreader.py]: Should these be `float32` or `float64`?

File: cudf/tests/test_csvreader.py (hunk 1 above)
Comment (randerzander) [reply cudf/tests/test_csvreader.py]: done in latest commit

File: cudf/tests/test_csvreader.py (hunk 2 above)
Comment (randerzander) [reply cudf/tests/test_csvreader.py]: Specified as float32 in latest commit. Should we have a separate test for float64?

**Comment by kkraus14** [comment]
LGTM, merging. Thanks @BradReesWork @randerzander!

**Pull request merged**